src/util/syscall_sandbox.cpp

```diff
@@ -592,6 +592,7 @@ class SeccompPolicyBuilder
         allowed_syscalls.insert(__NR_getcwd);          // get current working directory
         allowed_syscalls.insert(__NR_getdents);        // get directory entries
         allowed_syscalls.insert(__NR_getdents64);      // get directory entries
+        allowed_syscalls.insert(__NR_linkat);          // create relative to a directory file descriptor
```

Comment:
Tested on Ubuntu 22.04:
```suggestion
        allowed_syscalls.insert(__NR_inotify_rm_watch);// remove an existing watch from an inotify instance
        allowed_syscalls.insert(__NR_linkat);          // create relative to a directory file descriptor
```

Reply:
Done in #24690.